autoload/battery.vim

```diff
@@ -93,10 +93,10 @@ function! s:get_available_backend() abort
     return 'powershell'
   elseif battery#backend#linux#is_available()
     return 'linux'
+	elseif battery#backend#termux#is_available()
```

Comment:
You are using TAB here right? The correct indent here is TWO SPACES like:

```suggestion
  elseif battery#backend#termux#is_available()
```

Reply:
Ok